### PR TITLE
Configurable style, second attempt

### PR DESCRIPTION
**Pull request by moisesbr-dw**
Configurable style, as discussed in #15.
Two styles are defined: `default` and `symbols`.

This second attempt uses an idea borrowed from plugin `extlist`.
A folder named `style` contains the CSS definitions for each available style, using the **same** CSS classes.
The CSS file which DokuWiki searches for simply does not exist when the plugin is installed.

Only when needed, the style file (according to the plugin configuration) is copied to the place where DokuWiki looks for it:
  1. when the file that DokuWiki wants does not exist; or
  2. when the user have changed the configuration parameter to a different value.

Instead of `style.less`, we copy to `all.less`, which is also used for printing.

Contents:
  * Configuration option
  * Configuration in english, esperanto and portuguese
  * Code change in action plugin
  * Rewrite of old and new styles in LESS, inside a dedicated folder
  * New icons for style `symbols`

For example, the user can force a red background in questions, no matter which style is used, with the following code in `userstyle.css`:
```
div.dokuwiki div.qna-question {
  background-color: #f00 !important;
}
```

### CODE REVIEW

File: style/symbols.less [hunk 1]
```diff
@@ -0,0 +1,64 @@
+div.dokuwiki {
+  div.qna-toc {
+    & > ul {
+      background-color: @ini_background_alt;
```
Comment (dwp-forge) [inline style/symbols.less]: Was this intended that way? IMO, this background looks rather odd on TOC.

**Comment by moisesbr-dw** [comment]
OK, now the TOC does not have background nor border in Symbols style.

**Comment by moisesbr-dw** [comment]
One more style: `memes`.
As expected, only configuration was changed; code remained the same.

Screenshots (right side clipped):

image

image

**Comment by dwp-forge** [comment]
Awesome! :)

**Comment by moisesbr-dw** [comment]
Thanks.
Now it's your turn ;-)

**Comment by dwp-forge** [comment]
Well, I had some doubts when reviewing action part of c5e7f53, but after considering different options decided not to comment on that. After all how likely is that two different styles will have definition files of exactly the same size.

Guess what? ;)

**Comment by moisesbr-dw** [comment]
> Well, I had some doubts when reviewing action part of c5e7f53, but after considering different options decided not to comment on that.

Yeah, a dirt ugly hack.
But I think it is the simplest and most efficient way without inspecting the file.

> After all how likely is that two different styles will have definition files of exactly the same size. Guess what? ;)

Hate you, Murphy!!!

image

**Comment by moisesbr-dw** [comment]
I hope it is ready now. Phew!

**Comment by moisesbr-dw** [comment]
If you want to commit all 4 commits in one shot, here's how:
https://help.github.com/articles/configuring-commit-squashing-for-pull-requests/

**Comment by dwp-forge** [comment]
Finally :)

Thanks a lot!

**Pull request merged**